app/routes/send-endringsmelding.tsx

```diff
@@ -1,19 +1,113 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import TekstBlokk from '~/komponenter/tekstblokk/TekstBlokk';
 import { ESanitySteg } from '~/typer/sanity/sanity';
 import { TypografiTyper } from '~/typer/typografi';
-import { useTekster } from '~/hooks/contextHooks';
+import { useSpråk, useTekster } from '~/hooks/contextHooks';
 import InnholdKonteiner from '~/komponenter/innholdkonteiner/InnholdKonteiner';
+import css from './send-endringsmelding.module.css';
+import cssFritekst from './fritekstfelt.module.css';
+
+import { Button, Textarea } from '@navikt/ds-react';
+import { hentPathForSteg } from '~/utils/hentPathForSteg';
+import { useNavigate } from '@remix-run/react';
 
 export default function SendEndringsmelding() {
   const tekster = useTekster(ESanitySteg.SEND_ENDRINGER);
```

Comment:
```suggestion
  const { overskrift, fritekstfeltTittel, ...... , fritekstfeltBeskrivelse} = useTekster(ESanitySteg.SEND_ENDRINGER);
```
Her kan man destrukturere ut de feltene man har behov for, og bruke de direkte